### PR TITLE
Add NumberInput interface specification

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: NumberInput/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| NumberInput | c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401  |
+
+
+## Description
+
+Interface for number input.
+
+## Functions
+
+`int` - prints prompt message to the user and returns singed integer entered by user
```
Comment (Keshoid) [inline NumberInput/README.md]: Lets use `select` in all input interfaces.

File: NumberInput/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	min: int256
+	max: int256
+
+returns:
+
+	{ value: int256 }
+
+## Declaration in Solidity
+
+TODO: Add declaration
```
Comment (Keshoid) [inline NumberInput/README.md]:
```suggestion
interface INumberInput {

	function select(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);

}
```

File: NumberInput/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Declaration in C++
+
+TODO: Add declaration
```
Comment (Keshoid) [inline NumberInput/README.md]:
```suggestion
namespace tvm { namespace schema {

__interface INumberInput {
	[[internal, answer_id]]
	int256 select(string prompt, int256 min, int256 max);
}

}};
```

File: NumberInput/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Code Example
+
+### Solidity
+
+TODO: Add example
```
Comment (Keshoid) [inline NumberInput/README.md]:
```suggestion
Example.sol
```

**Review by Keshoid — CHANGES_REQUESTED** [review]
[number_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5970330/number_input.tar.gz)
NumberInput.sol
NumberInput.hpp
NumberInput.abi.json

File: NumberInput/README.md (hunk 1 above)
Comment (BigRoma) [reply NumberInput/README.md]:
```suggestion
`get` - prints prompt message to the user and returns singed integer entered by user
```

File: NumberInput/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+}
+
+## Declaration in C++
+
+namespace tvm { namespace schema {
+
+__interface INumberInput {
+	[[internal, answer_id]]
+	int256 select(string prompt, int256 min, int256 max);
```
Comment (BigRoma) [inline NumberInput/README.md]:
```suggestion
	int256 get(string prompt, int256 min, int256 max);
```

File: NumberInput/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+returns:
+
+	{ value: int256 }
+
+## Declaration in Solidity
+
+interface INumberInput {
+
+	function select(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);
```
Comment (BigRoma) [inline NumberInput/README.md]:
```suggestion
	function get(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);
```

**Comment by BigRoma** [comment]
> [number_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5970330/number_input.tar.gz)
> NumberInput.sol
> NumberInput.hpp
> NumberInput.abi.json

@Keshoid Please, check it.

File: NumberInput/examples/NumberInput.sol [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+interface INumberInput {
+
+	function get(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);
+
+}
+
+library NumberInput {
+
+	uint256 constant ID = 0x05606e612a821f3ecf72621a28304c5ae5fcf7681637bf9aaf74954a190175b5;
```
Comment (Keshoid) [inline NumberInput/examples/NumberInput.sol]: Please, fix it to `c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401`

**Review by Keshoid — CHANGES_REQUESTED** [review]

File: NumberInput/examples/NumberInput.sol (hunk 7 above)
Comment (BigRoma) [reply NumberInput/examples/NumberInput.sol]: > Please, fix it to `c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401`

```suggestion
	uint256 constant ID = 0xc5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401;
```

**Review by sertony — APPROVED** [review]

File: NumberInput/README.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+## Functions
+
+`get` - prints prompt message to the user and returns singed integer entered by user
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+	min: int256
+	max: int256
```
Comment (Keshoid) [inline NumberInput/README.md]:
```suggestion
    answerId: uint32 - id of callback function.
    prompt: bytes - utf-8 string to print to the user before input.
    min: int256 - minimal value allowed to input.
    max: int256 - maximum value allowed to input. 
```

**Review by Keshoid — CHANGES_REQUESTED** [review]

**Review by Keshoid — CHANGES_REQUESTED** [review]
Please, update this files: changed `answerId` to `_answer_id`.
[number_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5980819/number_input.tar.gz)
NumberInput.hpp 
NumberInput.sol
NumberInput.abi.json
Example.sol

File: NumberInput/README.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+Interface for number input.
+
+## Functions
+
+`get` - prints prompt message to the user and returns singed integer entered by user
+
+arguments:
+
+    answerId: uint32 - id of callback function.
```
Comment (Keshoid) [inline NumberInput/README.md:21]:
```suggestion
    _answer_id: uint32 - id of callback function.
```

File: NumberInput/README.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+returns:
+
+	{ value: int256 }
+
+## Declaration in Solidity
+
+interface INumberInput {
+
+	function get(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);
```
Comment (Keshoid) [inline NumberInput/README.md:34]:
```suggestion
	function get(uint32 _answer_id, string prompt, int256 min, int256 max) external returns (int256 value);
```

Comment (sertony) [reply NumberInput/README.md:34]: mixed cases are ugly, also the starting `_` is weird, since it's not a private variable.

**Review by Keshoid — COMMENTED** [review]
[number_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5981353/number_input.tar.gz)

**Review by Keshoid — APPROVED** [review]

**Review by sertony — APPROVED** [review]

**Pull request merged**